### PR TITLE
chore: misc HIP-423 fixes

### PR DESCRIPTION
**Pull request by tinker-michaelj**
**Description**:
 - Cherry-pick #16988

### CODE REVIEW

File: hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/transactions/schedule/HapiScheduleCreate.java [hunk 1]
```diff
@@ -172,12 +172,12 @@ public HapiScheduleCreate<T> waitForExpiry(boolean value) {
 
     public HapiScheduleCreate<T> expiringAt(final long expiry) {
         this.longTermExpiry = expiry;
-        return waitForExpiry();
+        return this;
```
Comment (Neeharika-Sompalli) [inline hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/transactions/schedule/HapiScheduleCreate.java:175]: This file seem to be different from original PR. Is this intentional ?

**Review by Neeharika-Sompalli — APPROVED** [review]
Some test files are extra compared to original PR. LGTM! Thanks @tinker-michaelj

File: hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/transactions/schedule/HapiScheduleCreate.java (hunk 1 above)
Comment (tinker-michaelj) [reply hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/transactions/schedule/HapiScheduleCreate.java:175]: Yes Neeha, this is intentional.
 - The `release/0.57` branch didn't yet have all the tests I brought in from `develop`; and,
 - Some of those tests assume this behavior from `HapiScheduleCreate`.

**Review by iwsimon — APPROVED** [review]
LGTM, thanks @tinker-michaelj

**Pull request merged**